.pre-commit-config.yaml

```diff
@@ -1,8 +1,8 @@
 default_stages: [commit, push]
-exclude: ^src/bin/
+exclude: ^fact_helper_file/bin/
```

Comment:
Err, that does not make sense to me. Mind to explain?

Reply:
@jstucke ping

Reply:
hmm I guess that exclude just doesn't matter since the stuff in bin is not versioned by git anyway

Reply:
> Err, that does not make sense to me. Mind to explain?

The idea probably was to not lint scripts in the bin directory (but since they are not versioned by git, they are excluded anyway)

Reply:
I actually was referring to the fact that this says `fact_helper_file` instead `fact_extractor`.